sys/linux/dev_dri.txt.const

```diff
@@ -1,177 +1,183 @@
-DRM_IOCTL_PRIME_FD_TO_HANDLE = 3222037550
-DRM_IOCTL_PRIME_HANDLE_TO_FD = 3222037549
-DRM_IOCTL_RES_CTX = 3222299686, 386:arm:3221775398
-DRM_IOCTL_RM_CTX = 3221775393
-DRM_IOCTL_RM_MAP = 1076388891, 386:arm:1075340315, mips64le:ppc64le:2150130715
-DRM_IOCTL_SET_CLIENT_CAP = 1074816013, mips64le:ppc64le:2148557837
-DRM_IOCTL_SET_MASTER = 25630, mips64le:ppc64le:536896542
-DRM_IOCTL_SET_SAREA_CTX = 1074816028, 386:arm:1074291740, mips64le:ppc64le:2148557852
-DRM_IOCTL_SET_UNIQUE = 1074816016, 386:arm:1074291728, mips64le:ppc64le:2148557840
-DRM_IOCTL_SET_VERSION = 3222299655
-DRM_IOCTL_SG_ALLOC = 3222299704, 386:arm:3221775416
-DRM_IOCTL_SG_FREE = 1074816057, 386:arm:1074291769, mips64le:ppc64le:2148557881
-DRM_IOCTL_SWITCH_CTX = 1074291748, mips64le:ppc64le:2148033572
-DRM_IOCTL_UNLOCK = 1074291755, mips64le:ppc64le:2148033579
-DRM_IOCTL_VERSION = 3225445376, 386:arm:3223610368
-DRM_IOCTL_WAIT_VBLANK = 3222823994, 386:arm:3222299706
-DRM_MODE_ATOMIC_ALLOW_MODESET = 1024
-DRM_MODE_ATOMIC_NONBLOCK = 512
-DRM_MODE_ATOMIC_TEST_ONLY = 256
-DRM_MODE_CURSOR_BO = 1
-DRM_MODE_CURSOR_MOVE = 2
-DRM_MODE_FB_DIRTY_ANNOTATE_COPY = 1
-DRM_MODE_FB_DIRTY_ANNOTATE_FILL = 2
-DRM_MODE_FB_INTERLACED = 1
-DRM_MODE_FB_MODIFIERS = 2
-DRM_MODE_OBJECT_ANY = 0
-DRM_MODE_OBJECT_BLOB = 3149642683
-DRM_MODE_OBJECT_CONNECTOR = 3233857728
-DRM_MODE_OBJECT_CRTC = 3435973836
-DRM_MODE_OBJECT_ENCODER = 3772834016
-DRM_MODE_OBJECT_FB = 4227595259
-DRM_MODE_OBJECT_MODE = 3739147998
-DRM_MODE_OBJECT_PLANE = 4008636142
-DRM_MODE_OBJECT_PROPERTY = 2964369584
-DRM_MODE_PAGE_FLIP_ASYNC = 2
-DRM_MODE_PAGE_FLIP_EVENT = 1
-DRM_MODE_PAGE_FLIP_TARGET_ABSOLUTE = 4
-DRM_MODE_PAGE_FLIP_TARGET_RELATIVE = 8
-DRM_PRIME_CAP_EXPORT = 2
-DRM_PRIME_CAP_IMPORT = 1
-DRM_PROP_NAME_LEN = 32
-DRM_RM_COMMAND = 1
-DRM_UNINST_HANDLER = 3
-O_CLOEXEC = 524288
-O_NONBLOCK = 2048, mips64le:128
-_DRM_AGP = 3
-_DRM_AGP_BUFFER = 2
-_DRM_CONSISTENT = 5
-_DRM_CONTAINS_LOCK = 32
-_DRM_CONTEXT_2DONLY = 2
-_DRM_CONTEXT_PRESERVED = 1
-_DRM_DMA_BLOCK = 1
-_DRM_DMA_LARGER_OK = 64
-_DRM_DMA_PRIORITY = 4
-_DRM_DMA_SMALLER_OK = 32
-_DRM_DMA_WAIT = 16
-_DRM_DMA_WHILE_LOCKED = 2
-_DRM_DRIVER = 128
-_DRM_FB_BUFFER = 8
-_DRM_FRAME_BUFFER = 0
-_DRM_HALT_ALL_QUEUES = 16
-_DRM_HALT_CUR_QUEUES = 32
-_DRM_KERNEL = 8
-_DRM_LOCKED = 4
-_DRM_LOCK_FLUSH = 4
-_DRM_LOCK_FLUSH_ALL = 8
-_DRM_LOCK_QUIESCENT = 2
-_DRM_LOCK_READY = 1
-_DRM_PAGE_ALIGN = 1
-_DRM_PCI_BUFFER_RO = 16
-_DRM_READ_ONLY = 2
-_DRM_REGISTERS = 1
-_DRM_REMOVABLE = 64
-_DRM_RESTRICTED = 1
-_DRM_SCATTER_GATHER = 4
-_DRM_SG_BUFFER = 4
-_DRM_SHM = 2
-_DRM_VBLANK_ABSOLUTE = 0
-_DRM_VBLANK_EVENT = 67108864
-_DRM_VBLANK_FLIP = 134217728
-_DRM_VBLANK_HIGH_CRTC_MASK = 62
-_DRM_VBLANK_NEXTONMISS = 268435456
-_DRM_VBLANK_RELATIVE = 1
-_DRM_VBLANK_SECONDARY = 536870912
-_DRM_VBLANK_SIGNAL = 1073741824
-_DRM_WRITE_COMBINING = 16
-__NR_ioctl = 54, amd64:16, arm64:riscv64:29, mips64le:5015
-__NR_openat = 56, 386:295, amd64:257, arm:322, mips64le:5247, ppc64le:286, s390x:288
+arches = arm64
```

Comment:
Const files need to be generated for all arches. Please run `make extract`.

Reply:
Fixed